### PR TITLE
[aws-datastore] Minor code style tweaks in SQLiteStorageAdapter

### PR DESCRIPTION
**Pull request by jamesonwilliams**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java [hunk 1]
```diff
@@ -732,16 +732,11 @@ private boolean dataExistsInSQLiteTable(
...
-                .append("*").append(SqlKeyword.DELIMITER)
-                .append(SqlKeyword.FROM).append(SqlKeyword.DELIMITER)
-                .append(Quotes.wrapInSingle(tableName)).append(SqlKeyword.DELIMITER)
-                .append(SqlKeyword.WHERE).append(SqlKeyword.DELIMITER)
-                .append(columnName).append(SqlKeyword.DELIMITER)
-                .append(SqlKeyword.EQUAL).append(SqlKeyword.DELIMITER)
-                .append(Quotes.wrapInSingle(columnValue))
-                .toString();
+        final String queryString = "" +
+            SqlKeyword.SELECT + SqlKeyword.DELIMITER + "*" + SqlKeyword.DELIMITER +
```
Comment (jpignata) [inline aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java:736]: Outside of the bounds of this PR, but we don't need to actually transit data from SQLite here. You could change `"*"` to `"1"` given all we are doing is checking the count of returned rows in the cursor.

File: aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java [hunk 2]
```diff
@@ -785,11 +774,6 @@ private Completable updateModels() {
         return gson.getAdapter(itemClass).fromJson(modelInJsonFormat);
     }
 
-    @VisibleForTesting
-    Cursor getQueryAllCursor(@NonNull String tableName) throws DataStoreException {
```
Comment (jpignata) [inline aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java:789]: Unused?

Comment (jamesonwilliams) [reply aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java:789]: 'Twas, 'twas

File: aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java (hunk 1 above)
Comment (jamesonwilliams) [reply aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapter.java:736]: Aha! TIL https://stackoverflow.com/a/7171072/695787

Let's do it in a separate PR

**Pull request merged**